api/services/BedCountService.js

```diff
@@ -102,7 +102,7 @@ const handleReinstatements = (centre, tester) => {
   return centre;
 };
 
-const populateOOC = (centre) =>
+const oldPopulateOOC = (centre) =>
```

Comment:
could do with better naming than this, or put the `getOOCByCentreGroupByGenderAndReason` in to the start of the `populateOOC` and do away with it entirely